### ISSUE TITLE
fetchSoldiers 관련 에러 보고

### ISSUE THREAD

**Issue by ytw0728** [parksb/the-camp-lib#21]
fetchSoldiers에서 요청하고 있는 `main/cafeCreateCheckA.do`의 동작이 변경된 듯 합니다.

현재, 가입되어 있어도 0000 code가 반환되며 교육생 리스트를 전달해주지 않고 있는 듯 합니다.

해서 fetchSoldiers를 사용하지 않고, 교육부대/교육대 코드를 직접 사이트에서 수집해 적용하여 정상 동작하는 것을 확인했으니,
참고 부탁드립니다!

감사합니다.

**Comment by minseolee** [comment]
> fetchSoldiers에서 요청하고 있는 `main/cafeCreateCheckA.do`의 동작이 변경된 듯 합니다.
> 
> 현재, 가입되어 있어도 0000 code가 반환되며 교육생 리스트를 전달해주지 않고 있는 듯 합니다.
> 
> 해서 fetchSoldiers를 사용하지 않고, 교육부대/교육대 코드를 직접 사이트에서 수집해 적용하여 정상 동작하는 것을 확인했으니, 참고 부탁드립니다!
> 
> 감사합니다.

안녕하세요? 
혹시 traineeMgrSeq 를 말씀하시는 것일까요?

**Comment by ybbarng** [comment]
네 traineeMgrSeq가 맞습니다. 이 정보만 있으면 편지를 보낼 수 있는데, 기존에는 cafeCreateCheckA.do 여기서 얻을 수 있었던 정보이지만 이제는 얻지 못하는 것 같습니다. 사이트에서 이 값을 얻어서 환경 변수로 넣어주는 형태로는 편지를 전송할 수 있습니다.
